app/bundles/CoreBundle/ErrorHandler/ErrorHandler.php

```diff
@@ -491,18 +491,50 @@ function ($matches) {
                 if (!empty($error['showExceptionMessage'])) {
                     $message = $error['message'];
```

Comment:
This var and the one on line 489 is unused now.

Reply:
I re-ordered this part of code